src/traceme.c

```diff
@@ -375,12 +375,15 @@ tracer_thread(void *arg)
         goto clean;
     }
 
-    // Copy out arguments for the poll loop.
+    /*
+     * Copy arguments for the poll loop, as when we resume the parent thread,
```

Comment:
This should be `//` commented (so it'll become 2 lines instead of 4).

Reply:
Eh? So we can't use multi-line comments?

Reply:
Two lines of `//` is much easier to read than 4 lines of `/*...*/`.

Reply:
I'm confused. We have multi-line comments all over. What is it about this one that is special?

Reply:
It's not about allowing multiline comments. It's about freeing ourselves from some of pre-C99 ugliness. Which is easier to read overall:

```C
/*
 * blah
 * blah
*/
```

or:

```C
// blah
// blah
```

Vertical space is at a premium, especially with the move to widescreen monitors, so the latter is preferable. I can just about tolerate the former use case before function definitions, even though it's a bit archaic, but within functions, let's just use `//`. [Another way of asking this is: would you use `/*...*/` for a two line comment in Rust? I reckon not.]

As to why this one: I noticed it. I'd happily never see `/* ... */` in a PR ever again ;)

Reply:
I don't mind either.

If you really hate `/* */` though, then let's kill them all. At least then I know what style to use.